### DISCUSSION TITLE
Welcome to toggl-cli Discussions!

### DISCUSSION THREAD

**Discussion by shantanuraj** [watercooler-labs/toggl-cli#67] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by dmitrybaranov** [comment]
Many thanks for the extremely useful tool! I plugged that it into my claude+obsidian life management process and I'm totally happy! Cli is more useful than mcp for selfhosted apps.
